setup.py

```diff
@@ -174,6 +174,10 @@ def _run_cmake(self, ext):
                 [cmake, '--build', '.', '--target', 'install'] + build_args,
                 cwd=self.build_temp, env=env
             )
+            subprocess.check_call(
+                [ext.cmake_install_prefix+'/bin/neurondemo', '-nopython', '-nogui', '-c', 'quit()'],
+                cwd=self.build_temp, env=env
+            )
```

Comment:
@nrnhines : we were discussing this with Alex and Fernando, isn't it possible to embed logic of ` packaging/python/demo_update_libnrnmech.sh` here in the setup.py? It would be better to have all logic under setup.py instead of external script and to avoid expanding/re-building zip.

Reply:
That would be nice. I was unable to figure it out. As the problem was associated with a <failure of> delocate in ```packaging/python/build_wheels.bash``` that is where I fixed it. I think the choice is to let this languish til an elegant fix is developed (with the consequence of distributing broken wheels) or letting this be merged to the master until something better comes along. I was very hopeful for the -L option for delocate, but those hopes were dashed.

Reply:
> I was unable to figure it out.

Your `subprocess.check_call` is going to prepare neurondemo and then later `setup.py` is going to collect files from build directory to create wheel. So right after `subprocess.check_call` i.e. line 181 , we can execute the commands you have in the shell script. That more or less should do the job?

Reply:
Merging as is but will start another pull request to do the repair in ```setup.py``` instead of after delocate in ```packaging/python/build_wheels.bash```